tests/main/clients/operations/expected_client/custom_operations.py

```diff
@@ -0,0 +1,64 @@
+cQuery_GQL = """
+query cQuery {
+  constQuery
+}
+"""
+cMutation_GQL = """
+mutation cMutation {
+  constMutation
+}
+"""
+cSubscription_GQL = """
+subscription cSubscription {
+  constSubscription
+}
+"""
+getA_GQL = """
+query getA {
+  a {
+    value
+    valueB {
+      value
+    }
+  }
+}
+"""
+getAWithFragment_GQL = """
+query getAWithFragment {
+  a {
+    value
+    valueB {
+      ...fragmentB
+    }
+  }
+}
+
+fragment fragmentB on TypeB {
+  value
+}
+"""
+getXYZ_GQL = """
+query getXYZ {
+  xyz {
+    __typename
+    ... on TypeX {
+      valueX
+    }
+    ... on TypeY {
+      ...fragmentY
+    }
+  }
+}
+
+fragment fragmentY on TypeY {
+  valueY
+}
+"""
+__all__ = [
```

Comment:
Can we put `__all__` on top of the generated file so its first thing people see?